src/classes/UTIL_LightningComponentSupport.cls

```diff
@@ -36,23 +36,43 @@
 
 public with sharing class UTIL_LightningComponentSupport {
 
+    public static final String LIGHTNING_FIELD_TYPE_NUMBER = 'number';
```

Comment:
Let's rename these to be a little more clear about these being a data type, eg: "LIGHTNING_DATA_TYPE_NUMBER"